opentracing/tracer.py

```diff
@@ -82,3 +83,24 @@ def close(self):
         fut = Future()
         fut.set_result(True)
         return fut
+
+    def implementation_id(self):
+        """Returns the ImplementationID for this OpenTracing implementation.
+
+        :return: An ImplementationID instance that describes this OpenTracing
+            implementation.
+        """
+        return ImplementationID('noop', '1.0.0')
+
+
+# Per collections.namedtuple, the below defines a class called ImplementationID
+# which can be initialized via positional parameters or a kwargs-style
+# initializer.
+#
+#     dapper_impl = ImplementationID('dapper', '0.1.2')
+#     zipkin_impl = ImplementationID(name='zipkin', version='0.3.4')
+#
+ImplementationID = namedtuple('ImplementationID', 'name version')
```

Comment:
I like `namedtuple` (makes me feel like I'm in a language that's not python :), but I could be talked into returning an ad hoc string map, too.